static/app/components/updatedEmptyState.tsx

```diff
@@ -86,9 +65,9 @@ export default function UpdatedEmptyState({project}: {project?: Project}) {
     replayOptions: {block: true, mask: true},
   };
 
-  const install = docs.onboarding.install(docParams)[0];
-  const configure = docs.onboarding.configure(docParams);
-  const verify = docs.onboarding.verify(docParams);
+  const install = loadGettingStarted.docs.onboarding.install(docParams)[0];
+  const configure = loadGettingStarted.docs.onboarding.configure(docParams);
+  const verify = loadGettingStarted.docs.onboarding.verify(docParams);
```

Comment:
doing the destructuring means we can clean this up to be something like this:

```suggestion
  const {install, configure, verify} = docs.onboarding;
  const install = install(docParams)[0];
  const configure = configure(docParams);
  const verify = verify(docParams);
```